### ISSUE TITLE
kronometrix rc scripts must support status

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#39]
On Linux, RHEL, CentOS we have seen the rc scripts, kronometrix and kronometrix_transport do not support status option. We need to have that present.

**Comment by sparvu** [comment]
Will be fixed in next release 1.0.14:

```
[root@rh664 ~]# /etc/init.d/kronometrix status
Status Kronometrix recorders
 sysrec: running, pid: 2189
 cpurec: running, pid: 2207
 diskrec: running, pid: 2225
 nicrec: running, pid: 2243
 hdwrec: running, pid: 2261

or
 
[root@rh664 ~]# service kronometrix status
Status Kronometrix recorders
 sysrec: running, pid: 2189
 cpurec: running, pid: 2207
 diskrec: running, pid: 2225
 nicrec: running, pid: 2243
 hdwrec: running, pid: 2261
```

 * And transport:

```
[root@rh664 ~]# /etc/init.d/kronometrix_transport status
Status Kronometrix transport
 sender: running, pid: 9148

or

[root@rh664 ~]# service kronometrix_transport status
Status Kronometrix transport
 sender: running, pid: 9148
```

**Comment by sparvu** [comment]
Will make this permanent on Linux. For FreeBSD I will add the same option for rc.d daemon.
